projects/swimlane/ngx-ui/src/lib/components/select/select-dropdown.component.ts

```diff
@@ -250,6 +250,12 @@ export class SelectDropdownComponent implements AfterViewInit {
     }
   }
 
+  //  tab has to focus on the input field of filterable dropdown as earlier it was losing focus and selecting next dropdown
```

Comment:
When the filterable select input has focus, tab event opens the next popover if the next control is ngx-select.